### PR TITLE
feat(grid3d): prop to show-hide grid lines added

### PR DESCRIPTION
**Pull request by LeonidPolukhin**
The prop "gridLines" added to Grid3d layer.

### CODE REVIEW

**Review by hkfb — APPROVED** [review]

File: typescript/packages/subsurface-viewer/src/layers/grid3d/grid3dLayer.ts [hunk 1]
```diff
@@ -103,6 +103,11 @@ export interface Grid3DLayerProps extends ExtendedLayerProps {
      */
     colorMapFunction?: colorMapFunctionType | false;
 
+    /** Enable lines around cell faces.
+     *  default: true.
+     */
+    gridLines: boolean;
```
Comment (hkfb) [inline typescript/packages/subsurface-viewer/src/layers/grid3d/grid3dLayer.ts:109]: Suggest making `gridLines` optional with the `?` operator

Comment (LeonidPolukhin) [reply typescript/packages/subsurface-viewer/src/layers/grid3d/grid3dLayer.ts:109]: The same prop isn't optional in triangle layer. For consistency, should all the props of the kind be optional?

Comment (hkfb) [reply typescript/packages/subsurface-viewer/src/layers/grid3d/grid3dLayer.ts:109]: I think so, yes

Comment (LeonidPolukhin) [reply typescript/packages/subsurface-viewer/src/layers/grid3d/grid3dLayer.ts:109]: As it touches several layers suggest to make it in a separate task, if you don't mind.

**Pull request merged**